client/app/scripts/superdesk-workspace/workspace.js

```diff
@@ -324,6 +324,36 @@
                         superdeskFlags.flags.hideMonitoring = false;
                     }
                 };
+
+                /*
+                 * By using keyboard shortcuts, change the current showed view
+                 *
+                 */
+                var opt = {global: true};
+                keyboardManager.bind('alt+h', function (e) {
```

Comment:
we were using `ctrl` + something shortcuts, so not sure we should use `alt` for these, what do you think @fritzSF ?